src/test/java/ru/stepev/test/training/at/hw2/model/Page.java

```diff
@@ -0,0 +1,70 @@
+package ru.stepev.test.training.at.hw2.model;
+
+import java.util.Arrays;
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class Page {
+
+    private  String expectedUserName = "ROMAN IOVLEV";
+    private  String pageURL = "https://jdi-testing.github.io/jdi-light/index.html";
+    private  String expectedTitle = "Home Page";
+    private  String userName = "Roman";
+    private  String password = "Jdi1234";
+    private  String loginLinkXpath = "//a[@href='#']";
+    private  String headerMenuItemsXpath =
+        "//ul[@class='uui-navigation nav navbar-nav m-l8']//following-sibling::a";
+    private  String leftSideMenuItemsXpath =
+        "//ul[@class='sidebar-menu left']//following-sibling::a";
```

Comment:
Looks like some mixing of the **locators**, **test data** and **expected data**
Could you please refactor this class to split into independent categories